src/main/java/org/cyclops/everlastingabilities/item/ItemGuiAbilityContainer.java

```diff
@@ -53,15 +53,30 @@ protected ItemGuiAbilityContainer(ExtendedConfig eConfig) {
     public void addInformation(ItemStack itemStack, EntityPlayer entityPlayer, List list, boolean par4) {
         super.addInformation(itemStack, entityPlayer, list, par4);
         IAbilityStore abilityStore = itemStack.getCapability(MutableAbilityStoreConfig.CAPABILITY, null);
-        boolean empty = true;
-        for (Ability ability : abilityStore.getAbilities()) {
-            empty = false;
-            String name = L10NHelpers.localize(ability.getAbilityType().getUnlocalizedName());
-            list.add(TextFormatting.YELLOW + name + ": " + TextFormatting.RESET + ability.getLevel());
-        }
-        if (empty) {
-            list.add(TextFormatting.GRAY.toString() + TextFormatting.ITALIC + L10NHelpers.localize("general.everlastingabilities.empty"));
+
+        switch(abilityStore.getDisplayType()) {
```

Comment:
I don't really like switch cases.
What about delegating to an `addInformation` method to DisplayType and moving this logic in there?
This will also make it safer in the future when other display types would be added, that they automatically will require `addInformation` to be implemented.